### PR TITLE
PLAT-64979 Update c3 c3-nginx-ingress chart to export health port

### PR DESCRIPTION
**Pull request by c3-pranjaysagar**

### CODE REVIEW

File: charts/ingress-nginx/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 annotations:
   artifacthub.io/changes: |
-     - "[helm] Support custom port configuration for internal service (#9846)"
-     - "Adding resource type to default HPA configuration to resolve issues with Terraform helm chart usage (#9803)"
-     - "Update Ingress-Nginx version controller-v1.7.1"
+     - "Use controller-v1.7.1"
```
Comment (c3pradeepbista) [inline charts/ingress-nginx/Chart.yaml:3]: isn't this unrelated change?

**Comment by c3pradeepbista** [comment]
@c3-amitsalunke Please help us provide an environment with a cloud-armor enabled cluster in order to test this change.

**Review by c3pradeepbista — APPROVED** [review]

File: charts/ingress-nginx/Chart.yaml (hunk 1 above)
Comment (c3-pranjaysagar) [reply charts/ingress-nginx/Chart.yaml:3]: Yes,
These are the changes that opensource community has done in this version I believe, do we need to preserver that information?
